docs/config.adoc

```diff
@@ -53,11 +53,18 @@ It's possible to configure open-city-profile to communicate with a https://www.k
 
 == Audit events
 
-Audit events are produced using the https://docs.python.org/3/library/logging.html[Python logging module].
+Profile data access produces audit events. Audit events may be output to multiple destinations. The destinations can be enabled individually. By default all outputs are disabled.
 
-By default audit event production is disabled. In order to enable audit event production, set `AUDIT_LOGGING_ENABLED` to `True`.
+=== Database output
 
-By default audit event output is sent to `stdout`. It's possible to send the output to a file instead, by setting the `AUDIT_LOG_FILENAME` to the desired filename. The filename may be randomized by including capital "X" characters in it. The "X"s get replaced by random characters.
+- `AUDIT_LOG_TO_DB_ENABLED`: enable audit logging to database by setting to `True`. Default is `False`.
+
+=== Python logger output
+
+Output as JSON using the https://docs.python.org/3/library/logging.html[Python logging module].
+
+- `AUDIT_LOG_TO_LOGGER_ENABLED`: enable audit logging to logger by setting to `True`. Default is `False`.
```

Comment:
There was a feature flag before. I guess it was nowadays mostly for local development – you could choose whether audit logs were produced (to file or `stdout`) or not. The previous feature flag controlled the audit logging system as a whole. Now as there are two possible outputs for the logs, I changed the feature flags to only control the separate outputs and not the whole system.

It wasn't possible to fully control the logger output for audit logging before either. It's either `stdout`, file or nothing. Those options have remained the same. Turning off logger output saves the effort of serializing the audit log entries to JSON. Whether that's meaningful in any real situation I don't know. 

In the cloud environments, once audit logs are ingested to Elastic from the database, I think we don't want the logger output to file and I don't see a reason why we would need that to `stdout` either, so turning the logger output completely off is needed. One way or another. Or do we remove the whole logger output?

Reply:
I think it could be nice to remove the json logging altogether after the logs are in the database. But we can do it in the future, just like in the Keycloak.

Reply:
One option could be that we change the logger output more than is currently done in this PR. We could ditch the whole JSON serialization. There could be a possibility to get audit logs to `stdout`, in some format, perhaps for development purposes. But I guess we can't do that yet. We have to keep the JSON logger thing active until the ingesting of logs from database is working.